packages/website/components/alert.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import clsx from 'clsx'
+
+/**
+ * @typedef {Object} AlertProps
+ * @prop {string} [className]
+ * @prop {('top'|'bottom')} [position]
+ * @prop {('error'|'success'|'info')} [type]
+ * @prop {Number} [timer] How much seconds before onTimerEnd should be called
+ * @prop {Function} [onTimerEnd]
+ * @prop {JSX.Element} [children]
+ */
+
+const ANIMATION_DURATION = 400
+
+/**
+ *
+ * @param {AlertProps} props
+ * @returns
+ */
+export default function Alert({
+  className,
+  position = 'bottom',
```

Comment:
Can it be from the top?...cover the nav and make dismissable.

Reply:
Sure, it's not changed here, this is the component general definition, but I'll make the specific error we're adding here from the top :)

Reply:
![image](https://user-images.githubusercontent.com/24696635/134952395-3eaaa5ae-66d5-433c-8e18-71a0ab524654.png)
![image](https://user-images.githubusercontent.com/24696635/134952450-fd5d3895-127f-445f-b9a8-5b882bcbaa9c.png)

Looks good @alanshaw ?